app/src/main/AndroidManifest.xml

```diff
@@ -46,6 +46,7 @@
           inside their source set.
         -->
         <activity-alias
+            android:exported="true"
```

Comment:
nit: Here and below: Trying to reformat this will put the `exported` property below `name`.